README_ja.md

```
@@ -1,51 +1,60 @@
 # ExifFrame
 
-*[Read this in English](README.md)*
+[![CI](https://github.com/amemya/ExifFrame/actions/workflows/ci.yml/badge.svg)](https://github.com/amemya/ExifFrame/actions/workflows/ci.yml)
+[![codecov](https://codecov.io/gh/amemya/ExifFrame/branch/main/graph/badge.svg)](https://codecov.io/gh/amemya/ExifFrame)
+[![Release](https://img.shields.io/github/v/release/amemya/ExifFrame)](https://github.com/amemya/ExifFrame/releases)
+[![Go Version](https://img.shields.io/github/go-mod/go-version/amemya/ExifFrame)](https://github.com/amemya/ExifFrame)
+[![Wails](https://img.shields.io/badge/Wails-v3-red.svg)](https://v3alpha.wails.io/)
+[![Platform](https://img.shields.io/badge/Platform-macOS%20%7C%20Windows%20%7C%20Linux-lightgrey.svg)]()
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)